website/aws.erb

```diff
@@ -424,6 +424,15 @@
                     </ul>
                 </li>
 
+                <li<%= sidebar_current("docs-aws-resource-budgets-budget") %>>
```

Comment:
Up to now we did not include documentation in our patches. (same for website/docs/r/budgets_budget.html.markdown). in this way the diff is kept smaller.

Reply:
you are right. I am going to remove it

Reply:
Done!